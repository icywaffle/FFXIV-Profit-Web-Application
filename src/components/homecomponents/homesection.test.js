import React from "react"
import { shallow } from "enzyme"

import HomeSection from "./homesection"

describe("HomeSection", () => {
    it("should render with any given props", () => {
        // Shallow is a real unit test
        // There should be no children rendering
        // It only tests that single component
        const props = {
        }
        const component = shallow(<HomeSection {...props} />)

        expect(component).toMatchSnapshot()
    })


})