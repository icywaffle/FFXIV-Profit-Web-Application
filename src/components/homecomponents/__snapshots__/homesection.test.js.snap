// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeSection should render with any given props 1`] = `
<section
  uk-scrollspy="cls:uk-animation-fade"
>
  <div
    className="uk-container"
  >
    <div>
      <div />
      <div>
        <img />
      </div>
    </div>
  </div>
</section>
`;
